kitsune/search/tests/test_api.py

```diff
@@ -75,10 +75,13 @@ def test_invalid_locale(self):
         eq_(res.status_code, 400)
         eq_(res.data['detail'], {'locale': 'Could not find locale "bad-medicine".'})
 
-    def test_invalid_fallback_locale(self):
-        for locale, fallback in settings.NON_SUPPORTED_LOCALES.items():
-            if fallback is not None:
-                break
+    def test_invalid_fallback_locale_none_case(self):
+        # Test the locale -> locale case.
+        non_none_locale_fallback_pairs = [
+            (key, val) for key, val in sorted(settings.NON_SUPPORTED_LOCALES.items())
+            if val is not None
+        ]
+        locale, fallback = non_none_locale_fallback_pairs[0]
```

Comment:
It would, but if someone changes how that setting works, then this would be invalid and may or may not break appropriately.

This isn't really that complex. It's pretty straight-forward. If you want, I could code up `first` and then use that.